internal/rpm/rpm.go

```diff
@@ -0,0 +1,37 @@
+// Package rpm allows for inspecting RPM databases in BerkleyDB, NDB, and SQLite
+// formats.
+package rpm
+
+import (
+	"context"
+	"io"
+	"iter"
+)
+
+const Version = "10"
```

Comment:
confirming there are no changes to merit a version bump?

Reply:
Shouldn't be